docs/operator-manual/upgrading/2.3-2.4.md

```diff
@@ -24,18 +24,17 @@ The signature algorithm is negotiated with the SSH server when the connection is
 list of accepted signature algorithms, and if the server has a match, the connection proceeds. For most SSH servers on
 up-to-date git providers, acceptable algorithms other than `ssh-rsa` should be available.
 
-Before upgrading to Argo CD 2.4, check whether your git provider(s) using SSH authentication support algorithms newer
-than `rsa-ssh`.
+As long as `ssh_rsa` is not supported as single signature algorithm in Gitlab server, OpenSSH_8.9 is not mandatory on server side. Fallback to alternative OpenSSH version is acknowledged like this:
 
-1. Make sure your version of SSH >= 8.9 (the version used by Argo CD). If not, upgrade it before proceeding.
+```debug1: Local version string SSH-2.0-OpenSSH_8.9p1 Ubuntu-3
+debug1: Remote protocol version 2.0, remote software version OpenSSH_8.2p1 Ubuntu-4ubuntu0.5
+debug1: compat_banner: match: OpenSSH_8.2p1 Ubuntu-4ubuntu0.5 pat OpenSSH* compat 0x04000000
+```
 
-   ```shell
-   ssh -V
-   ```
-   
-   Example output: `OpenSSH_8.9p1 Ubuntu-3, OpenSSL 3.0.2 15 Mar 2022`
+Before upgrading to Argo CD 2.4, check whether your git provider(s) using SSH authentication support algorithms newer
+than `rsa-ssh`:
 
-2. Once you have a recent version of OpenSSH, follow the directions from the [OpenSSH 8.8 release notes](https://www.openssh.com/txt/release-8.7):
+   Follow the directions from the [OpenSSH 8.8 release notes](https://www.openssh.com/txt/release-8.7):
```

Comment:
Could this section be un-indented, since it is no longer part of a list?

Reply:
Agreed